### ISSUE TITLE
Previsão

### ISSUE THREAD

**Issue by diego-rapoport** [gjvnq/OpenICP-BR#2]
Existe alguma previsão de quando esse software estará disponível pra uso com validade jurídica? Utilizo muito o linux e ele me ajudaria imensamente no trabalho.

**Comment by gjvnq** [comment]
Não. Na verdade o projeto ficou parado por conta da minha faculdade.

**Comment by diego-rapoport** [comment]
Se eu manjasse mais de linguagens de baixo nível com certeza tentaria fazer contribuições. Poder assinar documentos no meu notebook seria bom demais.
